src/client/components/heroImage/buttonComponent/button.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
@varsha84 please remove your button component from this PR. You can have it locally to test but this component needs a separate PR that is being handled in this issue https://github.com/HackYourFuture-CPH/fp-class18/issues/2 

@simonreddy2001 please check this with @varsha84